generators/python/core_utilities/shared/force_multipart.py

```diff
@@ -1,12 +1,15 @@
-class ForceMultipartDict(dict):
+from typing import Any, Dict
+
+
+class ForceMultipartDict(Dict[str, Any]):
```

Comment:
str, Any is probs fine for our usage but I'd also be down to rip `KT` and `VT` generics